cms/jinja2/templates/components/streamfield/equation_block.html

```diff
@@ -0,0 +1,6 @@
+<div class="ons-u-db-no-js_disabled">
+    {{ value.equation }}
+</div>
+<noscript>
+    <div class="ons-u-ta-center">{{ value.svg | safe }}</div>
```

Comment:
Question: Could this be done by serving the SVG via an `img` tag, rather than baking it into the content directly? It's a lot of body bloat.

It also has the benefit of only trying to render it if it's actually used, which likely has some performance benefits.

Reply:
Personally I don't think the impact is that significant – gzipped it's probably just a few KBs of extra payload. However if necessary we could create a custom view which returns the SVG, for instance.

The browser won't attempt to render it unless JS is disabled due to the `<noscript>` tag (if that is what you were talking about).

Reply:
It won't render it on the page, but it'll still need to download and parse it. I guess it depends how complex the SVGs are. Small SVGs might be ok. In general I tend to prefer offloading that to a separate request even if it does only save a little page size.

Reply:
I understand where you are coming from. I'll raise this with others and see how much of a priority we see this to be 👍

Reply:
My vote is on keeping this as is.

Offloading it to a separate request adds enough complexity that we don't need.